website/content/docs/nia/tasks.mdx

```diff
@@ -158,16 +159,22 @@ task {
   condition "schedule" {
     cron = "* * * * Mon"
   }
-  source_input "services" {
+  module_input "services" {
     names = ["web", "db"]
   }
 }
 ```
 
-Below are the available options for source input types and how to configure them:
+Below are the available options for module input types and how to configure them:
 
-- [Services source input](/docs/nia/terraform-modules/#services-source-input): configure through [`task.services`](/docs/nia/configuration#services) (deprecated) or [`source_input "services"`](/docs/nia/configuration#services-source-input)
-- [Consul KV source input](/docs/nia/terraform-modules/#consul-kv-source-input): configure through [`source_input "consul-kv"`](/docs/nia/configuration#consul-kv-source-input)
+- [Services module input](/docs/nia/terraform-modules/#services-module-input):
+  - [`task.services`](/docs/nia/configuration#services) field (deprecated)
+  - [`module_input "services"`](/docs/nia/configuration#services-configure-input) block
+  - [`module_input "services"`](/docs/nia/configuration#services-configure-input) block
```

Comment:
you have this line twice

Reply:
Thanks for catching! I'll remove the second line